documentation/guides/general_setup.md

```diff
@@ -0,0 +1,230 @@
+# General setup guide
+
+This is an exhaustive list for new developers. Feel free to skip the steps that
+you have already done on your own.
+
+## Operating system
```

Comment:
I would add a note at the top of this section saying something along the lines of "Openverse development is currently supported for UNIX-type environments (Linux and MacOS)."

This at least summarizes the current state before launching into our caveats regarding Windows/WSL.

Reply:
Added the Windows unsupported message in f8d9d6d.